frontend/src/app/search/SearchForm.tsx

```diff
@@ -78,13 +78,15 @@ export function SearchForm({
           <div className="tablet:grid-col-8">
             <SearchResultsHeader
               formRef={formRef}
-              searchResultsLength={searchResults.pagination_info.total_records}
+              searchResultsLength={
+                searchResults?.pagination_info?.total_records
+              }
```

Comment:
since the error state passes null values initially, these need to have the optional chaining operator